model/Core/Properties/originalNamespaces.md

```diff
@@ -0,0 +1,20 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# namespaces
```

Comment:
```suggestion
# originalNamespaces
```

Reply:
I'd even consider using `originalNamespaceMap` instead of just `originalNamespaces`.

Reply:
Agree